{
  "verb": "get",
  "path": "/api/2.0/accounts/{account_id}/credentials/{credentials_id}",
  "description": "Gets a Databricks credential configuration object for an account, both specified by ID.",
  "parameters": {
    "account_id": {
      "in": "path",
      "required": true,
      "type": "uuid",
      "description": "."
    },
    "credentials_id": {
      "in": "path",
      "required": true,
      "type": "uuid",
      "description": "Databricks Account API credential configuration ID"
    }
  },
  "responses": {
    "200": {
      "credentials_id": {
        "description": "Databricks credential configuration ID."
      },
      "credentials_name": {
        "type": "string",
        "description": "The human-readable name of the credential configuration object."
      },
      "aws_credentials": {
        "description": "object"
      },
      "sts_role": {
        "description": "object"
      },
      "account_id": {
        "description": "The Databricks account ID that hosts the credential."
      },
      "creation_time": {
        "description": "Time in epoch milliseconds when the credential was created."
      }
    },
    "401": {
      "error_code": "UNAUTHORIZED",
      "description": "The request does not have valid authentication credentials for the operation."
    },
    "404": {
      "error_code": "NOT_FOUND",
      "description": "Operation was performed on a resource that does not exist."
    },
    "500": {
      "error_code": "INTERNAL_SERVER_ERROR",
      "description": "Internal error."
    }
  }
}